api/routers/edr.py

```diff
@@ -112,6 +114,16 @@ async def get_locations(
     # Check for multiple coordinates or names on one station
     for station_id in platform_parameters.keys():
         if len(platform_coordinates[station_id]) > 1:
+            if (
+                calculate_largest_postition_deviation(platform_coordinates[station_id]) < 1e-4
+            ):  # all coordinates is with 10M
+                platform_coordinates[station_id] = {
+                    sorted(
+                        [i for i in platform_coordinates[station_id]],
+                        key=lambda x: (len(str(x[0])), len(str(x[1])), x[0], x[1]),
+                    )[-1]
+                }
+                continue
```

Comment:
Can you rewrite the flow, the continue here feels strange.
Might also be wrong, don't we need to still check the platform_names in this case?

Reply:
Yes, in does skip a needed test. I have updated it, so it can be reached.